site/content/docs/0.0/migration.md

```diff
@@ -159,6 +238,7 @@ From now on, by default, OUDS Web won't embed Bootstrap elements (like helpers,
   + @import "tokens/raw";
   + @import "tokens/semantic";
   + @import "tokens/component";
+    @import "ouds-maps";
```

Comment:
Hmm, I'd say so. We're not supposed to modify the migration guide from a previous version (except if it was wrong). Since `ouds-maps` wasn't there in 0.0.3, we shouldn't mention it in the 0.0.3 section.